control-plane/stor-port/src/types/v0/store/volume.rs

```diff
@@ -166,6 +166,9 @@ pub struct VolumeSpec {
     /// The publish context of the volume.
     #[serde(default)]
     pub publish_context: Option<HashMap<String, String>>,
+    #[serde(default)]
+    /// Volume Group related information.
```

Comment:
```suggestion
    /// Volume Group related information.
    #[serde(default)]
```
simply for consistency

Reply:
Right.